src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,74 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private final K[] keysArray;
+    private final V[] valuesArray;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
no need to initialize primitives with their default values